src/llvm/lib/Target/MSP430/MSP430ISelLowering.cpp

```diff
@@ -940,18 +940,7 @@ SDValue MSP430TargetLowering::LowerShifts(SDValue Op,
 
   // Expand non-constant shifts to loops:
```

Comment:
Is it performed by generic code now?

Reply:
The generated code stays the same. For example, `ashr16` from `test/CodeGen/MSP430/shifts.ll`:
```
ashr16:                                 ; @ashr16
; %bb.0:                                ; %entry
        cmp.b   #0, r13
        jeq     .LBB4_2
.LBB4_1:                                ; %entry
                                        ; =>This Inner Loop Header: Depth=1
        rra     r12
        sub.b   #1, r13
        jne     .LBB4_1
.LBB4_2:                                ; %entry
        ret
```